### DISCUSSION TITLE
Welcome to ExtractThinker Discussions!

### DISCUSSION THREAD

**Discussion by enoch3712** [enoch3712/ExtractThinker#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mophilly** [comment]
Hi. Long time database application designer and builder. Looking for a document workflow for a specific document set.

**Reply by enoch3712** [reply]
Hello! 

Sure! The idea is to be able to be fully flexible. Do you have any idea what you want to do? Maybe i can give you a working example
